.github/workflows/document-compilation.yml

```diff
@@ -32,3 +32,10 @@ jobs:
         uses: actions/upload-artifact@v2
```

Comment:
Do we still need the upload step?

Reply:
I would keep it that you can check the zip before creating a release.

Reply:
Usually this should not be needed. This is the reason because I integrated the ZIP creation into cmake. Like this you can locally check that everything is correct in the ZIP. I think we should remove this unneeded step.

Reply:
Thanks. Very nive improvement overall 👍